geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/JdbcConnectorServiceImpl.java

```diff
@@ -62,6 +64,18 @@ public void replaceRegionMapping(RegionMapping alteredMapping)
     mappingsByRegion.put(existingMapping.getRegionName(), alteredMapping);
   }
 
+  @Override
+  public boolean isMappingSynchronous(RegionMapping mapping, Cache cache) {
```

Comment:
I think it would be better to pass the region name in as a String instead of a RegionMapping. I see other methods on this class that take a region name like "getMappingForRegion".